test-site/package.json

```diff
@@ -26,12 +26,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
```

Comment:
now that this config was deleted, I'm sometimes seeing an error like this in the test-site:
`ERROR in Plugin "react" was conflicted between "../.eslintrc.js » @yext/eslint-config-slapshot/typescript-react » eslint-config-react-app » ../node_modules/eslint-config-react-app/base.js" and "BaseConfig » ../test-site/node_modules/eslint-config-react-app/base.js".`

have you gotten this too?

Reply:
I have not

Reply:
oh looks like re-compiling fixed it

Reply:
okay I just saw the error after a couple reinstall and reload of test-site. From the look of this [thread](https://stackoverflow.com/questions/70449712/create-react-app-error-in-plugin-react-was-conflicted-between-eslintrc-jso) and the error message, I think it's two versions of the react eslint plugin being used. I update test-site package.json to point eslint-config-react-app to the top level node_modules one. The issue is gone from my end now